noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -158,7 +155,7 @@ impl PublicContext {
             start_side_effect_counter: self.inputs.start_side_effect_counter,
             end_side_effect_counter: self.side_effect_counter,
             unencrypted_logs_hashes: self.unencrypted_logs_hashes.storage,
-            unencrypted_log_preimages_length: self.unencrypted_log_preimages_length + 4,
+            unencrypted_log_preimages_length: self.unencrypted_log_preimages_length,
```

Comment:
@AVM team: We found replicating the length of ts logs was both tricky and unnecessary for the core circuits. They are never actually checked/used outside of the circuits (the ts classes handle length, and correctly add 4 bytes where required for serialising to L1). No changes should be required for the AVM.